addons/mail/static/tests/mock_server/mock_models/ir_attachment.js

```diff
@@ -30,26 +30,29 @@ export class IrAttachment extends webModels.IrAttachment {
     }
 
     /** @param {number} ids */
-    _attachment_format(ids) {
+    _to_store(ids, store) {
         /** @type {import("mock_models").DiscussVoiceMetadata} */
         const DiscussVoiceMetadata = this.env["discuss.voice.metadata"];
 
-        return this.read(ids).map((attachment) => {
+        for (const attachment of this.browse(ids)) {
             const res = {
-                create_date: attachment.create_date,
                 checksum: attachment.checksum,
+                create_date: attachment.create_date,
                 filename: attachment.name,
                 id: attachment.id,
                 mimetype: attachment.mimetype,
                 name: attachment.name,
                 size: attachment.file_size,
+                thread:
+                    attachment.res_id && attachment.model !== "mail.composeMessage"
```

Comment:
typo: `mail.composer.message`

Reply:
Good catch, with `mail.composer.message` `res_id` should be 0 so the other part of the condition is sufficient, but it's better to still fix it for consistency.

Reply:
updated